lib/routes/jin10/category.ts

```diff
@@ -0,0 +1,225 @@
+import { Route, ViewType } from '@/types';
+import { getCurrentPath } from '@/utils/helpers';
+const __dirname = getCurrentPath(import.meta.url);
+
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { parseDate } from '@/utils/parse-date';
+import timezone from '@/utils/timezone';
+import { art } from '@/utils/render';
+import path from 'node:path';
+import { config } from '@/config';
+
+export const route: Route = {
+    path: '/category/:id',
+    categories: ['finance', 'popular'],
```

Comment:
New route won't be popular.
```suggestion
    categories: ['finance'],
```